### ISSUE TITLE
Topics for consideration for future versions

### ISSUE THREAD

**Issue by ArthurChapman** [gbif/doc-georeferencing-best-practices#51]
1. **Elevation accuracy for LIDAR** . See mention of accuracy of ~10cm RMSE in https://www.usgs.gov/faqs/what-lidar-data-and-where-can-i-download-it?qt-news_science_products=0#qt-news_science_products.  I am not sure what the horizontal accuracy of LIDAR is - perhaps need exploring - I know that the pixel resolution varies (1m, 5m, 8m?, finer?) Needs further exploration.

**Comment by deepreef** [comment]
I am working with a team of engineers who are very familiar with LIDAR technology.  If you have specific questions about limits/practices/capabilities of this technology, I can probably get answers.

**Comment by ArthurChapman** [comment]
Thanks @deepreef. At the moment we have just logged this as a topic we should consider adding when doing an update.  We will definitely call on you when we do something about this.  We should have included LIDAR in the sections on both vertical accuracy and horizontal accuracy but we unfortunately neglected this methodology when writing those sections.  I may contact you separately to chase up some information if I need it for a different project (a discussion I am having with the local council on flood mapping methodologies)

**Comment by ArthurChapman** [comment]
Some additional items for possible inclusion into a new Version:

1. **Dynamic Datums** (#2.6.4) - including International Terrestrial Reference System (ITRS) 2020 and the International Terrestrial Reference Frame 2020 (see for example: https://en.wikipedia.org/wiki/International_Terrestrial_Reference_System_and_Frame) and relationship to regional areas such as the Australian Terrestrial Reference Frame 2020 and implementation of Dynamic Datums such as GDA2020
2. **Vertical Datums** (#2.7.4)  new methods using SBAS for improving accuracy of Vertical Datums - e.g. the Australian Vertical Working Service (AVWS) (https://www.icsm.gov.au/australian-vertical-working-surface) 
3. **Smart Apps for Underwater Location** (#2.6.11) see https://divemagazine.com/scuba-diving-news/new-smart-app-for-underwater-gps

**Comment by tucotuco** [comment]
To add to the list, Voronoi diagrams for non-obvious boundaries.

**Comment by ArthurChapman** [comment]
This paper may be of interest and a good reference when we write something on LIDAR.

It has some interesting diagrams on differences between DTMs, DFMs and DSMs (Figure 1)

https://www.researchgate.net/publication/351469633_Airborne_LiDAR-Derived_Digital_Elevation_Model_for_Archaeology

**Comment by ArthurChapman** [comment]
A paper "Interpreting and Georeferencing the Concept of “Near” in Biodiversity Records" by PD Campbell, Biodiversity Informatics 18:13-23 2024 - https://doi.org/10.17161/qe3d5373 discusses the use of Veronoi diagrams for determining uncertainty for a range of vague locations. At very minimum, we should reference this paper, but maybe add another section @tucotuco

**Comment by tucotuco** [comment]
Agreed that a section is a good idea. Questions come up very oftern about vague situations that this provides a consistent method to determine.

**Comment by ArthurChapman** [comment]
In Section 2.5 4th line of 2nd Paragraph change 3 cases of epsg: to EPSG:

See discussion at https://github.com/tdwg/bdq/issues/307

**Comment by ArthurChapman** [comment]
epsg changed to EPSG throught document - See https://github.com/gbif/doc-georeferencing-best-practices/releases/tag/1.2.1. COMPLETED
